config.beta

```diff
@@ -9,3 +9,7 @@ export INDEXER_URL="https://github.com/algorand/indexer"
 export INDEXER_BRANCH="develop"
 export INDEXER_SHA=""
 export INDEXER_DISABLED=""
+export ALGOSEARCH_URL="https://github.com/kevguy/algosearch"
```

Comment:
@kevguy I've merged your PR in AlgoSearch repo, can you please update the `ALGOSEARCH_URL` and `ALGOSEARCH_BRANCH` in all the config files?